src/App.jsx

```diff
@@ -15,8 +15,8 @@ function App() {
     <Router>
```

Comment:
- [ ] [OPTIONAL] Your App is not working properly. when adding a new book the app crashes, this should be fixed in the previous steps, kindly fix this so your app is complete 

![Screenshot_20220723_154923](https://user-images.githubusercontent.com/25227022/180610291-63a6a72e-518f-4925-b6b9-e50051a41b08.png)